packages/components/src/components/TokenBalance/components/Logo.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Image } from '@chakra-ui/react';
+
+export const TokenLogo = ({ tokenContractAddress }) => (
+  <Image
+    src={`https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/${tokenContractAddress}/logo.png`}
+  />
+);
+
+//i.E. Dai
+//https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x6B175474E89094C44Da98b954EedeAC495271d0F/logo.png
```

Comment:
Can we move this component over to the `TokenBalance.tsx`? We don't need a separate file for this

Reply:
You can also remove those comments